labman/gui/templates/sequence_run_list.html

```diff
@@ -15,11 +15,15 @@
           "/sample_sheet' class='btn btn-success'>" +
           "<span class='glyphicon glyphicon-download'></span> " +
           "Download Sample Sheet</a>";
-        var preparationSheets = "<a href='/process/sequencing/" + row[5] +
-          "/preparation_sheets' class='btn btn-success'>" +
-          "<span class='glyphicon glyphicon-download'></span> " +
-          "Download Preparation Sheets</a>";
 
+        // ToDo: #327
+        var preparationSheets = '';
+        if (row[3] == 'Amplicon') {
```

Comment:
would it make sense to assign variable names for `row[3]` and `row[5]` to avoid magic numbers?